parser/src/main/scala/hmda/parser/fi/ts/TsCsvParser.scala

```diff
@@ -51,22 +58,31 @@ object TsCsvParser {
       |@| contact.success
     ) { TransmittalSheet }
 
+    println("there" + maybeTS)
+
     maybeTS.leftMap(_.toList).toEither
   }
 
   def toIntOrFail(value: String, fieldName: String): ValidationNel[String, Int] = {
-    convert(value.toInt, s"$fieldName is not an Integer")
+    println("calling toInt")
+    val c = convert(value.toInt, s"$fieldName is not an Integer")
+    println("val " + c)
+    c
   }
 
   def toLongOrFail(value: String, fieldName: String): ValidationNel[String, Long] = {
+    println("calling toLong")
     convert(value.toLong, s"$fieldName is not a Long")
   }
 
   private def convert[T](x: => T, message: String): ValidationNel[String, T] = {
-    Try(x) match {
-      case Failure(result) => message.failure.toValidationNel
-      case Success(result) => result.success
+    val t = Try(x) match {
+      case Failure(result) =>
+        println("failure " + message); message.failure.toValidationNel
+      case Success(result) => println("this is where it stops: " + result + ", " + result.getClass); val x = result.success; println("success! " + x); x
```

Comment:
Ok, so this is the bug, as near as I can figure:

When running the `HmdaFileParserSpec` test, it stops here on this line (at `result.success`).  However, if you run the `TsCsvParserSpec`, it passes in similar values and continues on with the test.